util/_create_hatchet_token_graphql.sh

```
@@ -2,4 +1,52 @@
+echo "[hatchet-token] Creating tenant token..."
+TOKEN=$(docker exec "${CONTAINER_NAME}" /hatchet-admin token create --config /config --tenant-id "${TENANT_ID}" | xargs)
+
+if [[ -z "${TOKEN}" ]]; then
+  echo "[hatchet-token] Failed to generate Hatchet token (empty response)." >&2
+  exit 1
+fi
+
+echo "[hatchet-token] Token generated successfully (prefix): ${TOKEN:0:12}..."
+
+# Write token into GraphQL package .env so Vitest (dotenv/config) can load it
+for d in packages/graphql; do
```

Shellcheck (0.10.0)

[warning] This loop will only ever run once. Bad quoting or missing glob/expansion?

(SC2043)